OpenRA.Mods.Common/Projectiles/NukeLaunch.cs

```diff
@@ -140,7 +140,7 @@ void Explode(World world, bool removeProjectile)
 			if (detonated)
 				return;
 
-			weapon.Impact(Target.FromPos(pos), firedBy.PlayerActor, Enumerable.Empty<int>());
+			weapon.Impact(Target.FromPos(pos), firedBy.PlayerActor);
```

Comment:
This crashes because `firedBy.PlayerActor` doesn't have a `CenterPosition`. Manually constructing the `WarheadArgs` the same way as `IonCannonPower` should fix the issue.